docs/MFA_ANALOGY.md

```diff
@@ -0,0 +1,88 @@
+# Analogy - An evening Bistro "Chez Rémy"
+
+Bistro "Chez Rémy" is an exclusive restaurant with one single kitchen station.
+
+## Terminology
+
+- `Coordinator`: **Head Chef** "_Rémy_" of "Chez Rémy"
+- `Session`: **An evening** at Bistro "Chez Rémy"
+- `Transactions (Requests)`: **Tables** in the dining area of "Chez Rémy"
+- `Accounts` (Entities): **Guests** at some table
+- `FactorSourceKind`: A **kind of chef**, Pastry (cumbersome tasks), Sauce (fast hard tasks), Grill (easy medium fast tasks)
+- `FactorSource`: A **chef** working at kitchen station an evening
+- `MatrixOfFactorInstances`: An **order of dishes** by some guest (potentially conditional)
+- `Signature` (Response): **Prepared dishes** by some chef
+- `Wallet User`: **Mother Nature**, a bad harvest can lead to tomato shortage, some chef might not be able to finish dishes due to lack of ingredients. It is good for mother nature if guests don't eat too much.
+
+## Story
+
+The Monday night of the August the 31st {`Session`} is a busy night at Bistro Chez Rémy, but the Head Chef _Rémy_ {`Coordinator`} is ready to direct his skilled chefs {`FactorSource`s}, all with different specialties {`FactorSourceKind`s}.
+
+- _Gus_ {`FactorSourceKind.ledger`}
+- _Jaq_ {`FactorSourceKind.arculus`}
+- _Stuart Little_ {FactorSource also of kind `.arculus`}
+- _Speedy Gonzales_ {`FactorSourceKind.device`}
+
+to prepare all dishes {`Signatures`} ordered by all guests {`Accounts`} at every table {`Transactions`}, ensuring all needed dishes are cooked for every order {`MatrixOfFactorInstances`}
+
+The kitchen of Bistro "Chez Rémy" is tiny, since the restaurant is placed in pricy Quartier Latin in the 5th arrondissement of Paris. Since the rent of the restaurant is so high, _Rémy_ chose to maximize space for the dining area; which has many large round tables with room for many quests, however, having allocated maximum space for dining area, the tiny kitchen has only one single kitchen station.
+
+Furthermore _Rémy_ devised some special constraints for how the restaurant works:
+
+- The guests {`Accounts`} must email their order of dishes to the restaurant in advance
+- No guest {`Account`} is allowed to modify their order or otherwise order more dishes when they are at the restaurant
+- Guest can specify "advanced" orders like "Ratatouille OR [Caprese Salad AND Spaghetti Bolognese]" {`MatrixOfFactorInstances`}
```

Comment:
In this case, Ratatouille would be the response for the override factor, right? Maybe helpful to indicate that.